flight_phase_files/HALO_RF12_20200211_info.yaml

```
@@ -321,7 +321,8 @@ segments:
 - kinds:
   - lidar_leg
   name: lidar leg
-  irregularities: [Roll angle not constant. Two small turns.]
+  irregularities:
+  - roll angle not constant; two small turns
```

maybe like this? These seem to be two independent irregularities, so maybe they should be treated as such.
```suggestion
  - roll angle not constant
  - two small turns
```